core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -322,6 +344,72 @@ public final String getBuildNowText() {
          */
         Map<TriggerDescriptor,Trigger<?>> getTriggers();
 
+        /**
+         * @deprecated use {@link #scheduleBuild(Cause)}
+         */
+        @Deprecated
+        @Override
+        default boolean scheduleBuild() {
+            return getParameterizedJobMixIn().scheduleBuild();
+        }
+
+        @Override
+        default boolean scheduleBuild(Cause c) {
+            return getParameterizedJobMixIn().scheduleBuild(c);
+        }
+
+        /**
+         * @deprecated use {@link #scheduleBuild(int, Cause)}
+         */
+        @Deprecated
+        @Override
+        default boolean scheduleBuild(int quietPeriod) {
+            return getParameterizedJobMixIn().scheduleBuild(quietPeriod);
+        }
+
+        @Override
+        default boolean scheduleBuild(int quietPeriod, Cause c) {
+            return getParameterizedJobMixIn().scheduleBuild(quietPeriod, c);
+        }
+
+        // omitting scheduleBuild2(int, Action...) since it is defined in SCMTriggerItem (could include less commonly used overloads if desired)
```

Comment:
Actually as per [this tip](http://stackoverflow.com/a/19976615/12916) it _could_ be defined here. Implementing classes would still need to define the method, but at least they could select a known `super` implementation, which is probably more discoverable than looking for similar methods on a mixin.